lib/main.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
     // convert Buffers before splitting into lines and processing
     src.toString().split('\n').forEach(function (line) {
       // matching "KEY' and 'VAL' in 'KEY=VAL'
-      var keyValueArr = line.match(/^\s*([\w\.\-]+)\s*=\s*(.*)?\s*$/)
+      var keyValueArr = line.match(/^\s*(?:export)?\s*([\w\.\-]+)\s*=\s*(.*)?\s*$/)
```

Comment:
@jcready @maxbeatty Oh duh, I can definitely make this fix and add tests very soon... please stand by.